src/main/java/kernbeisser/Windows/Inventory/InventoryController.java

```diff
@@ -110,9 +115,10 @@ public void print(InventoryReports selectedReport, boolean selected, boolean out
         report = new InventoryShelfDetails(shelves, inventoryDate);
         break;
       case COUNTINGLISTS:
-        // abfrage ob Datum der Zählliste in Vergangenheit
-        if (inventoryDate.isBefore(ChronoLocalDate.from(LocalDate.now().atStartOfDay())) &&
-                !getView().confirmPrint()) {
+        // Abfrage beim User, wenn Datum der Zaehlliste in Vergangenheit
+        String confirmMessage = createConfirmMessage(inventoryDate);
```

Comment:
confirmMessage wird nur gebraucht, wenn der folgende if true ist. also besser 
if (...) {
   String confirmMessage...;
   if (!getView().confirmPrint...) {return}
   oder inventoryDate als parameter an confirmPrint und createConfirmMessage in den View (oder ist das pingelig?)

Reply:
Zur Zeit werden alle Messages im View erstellt. Aber der Controller wäre der bessere Ort, wenn irgendwann der View ausgetauscht werden soll. Daher im Controller lassen.

Reply:
OK, wie du meinst. *Ich* hätte es aus Konsistenzgründen in die View gelegt.